index.js

```diff
@@ -1,19 +1,22 @@
-// Polyfill for `globalThis`
-const _globalThis = (() => {
-	if (typeof self !== 'undefined') {
-		return self;
-	}
-
+const getGlobal = property => {
 	/* istanbul ignore next */
-	if (typeof window !== 'undefined') {
-		return window;
+	if (typeof self !== 'undefined' && self && property in self) {
+		return self[property];
 	}
 
 	/* istanbul ignore next */
-	if (typeof global !== 'undefined') {
-		return global;
+	if (typeof window !== 'undefined' && window && property in window) {
+		return window[property];
 	}
-})();
+
+	return global[property];
+};
+
+const document = getGlobal('document');
+const URL = getGlobal('URL');
+const URLSearchParams = getGlobal('URLSearchParams');
```

Comment:
I'm targeting Node.js 10 now, so we can just use `URL` and `URLSearchParams` globals directly in the source. https://github.com/sindresorhus/ky/commit/c5771ee799a9dd8ad8c705262e028f6da19163ee